### PR TITLE
CreateIndependentOutfitter,'fits[content/incomplete]

### PR DESCRIPTION
**Pull request by LocalGod79**
replacing #1723

### CODE REVIEW

**Comment by Pointedstick** [comment]
Perhaps I'm being thick, but after reading the descriptions of the two new outfits, it's still not clear to me what they actually do.

**Comment by BorkBorkGoesTheCode** [comment]
I think they are supposed to be used as space brakes.

**Comment by LocalGod79** [comment]
it's difficult to be more concise than "station holding" or "Full Stop", but @BorkBorkGoesTheCode has the right idea.

basically, (for ages) people have been asking for a way to come to a "complete stop" within a system - these will outfits will allow them to come imperceptibly close (without a new key binding - a solution that raises issues in a possible controller port).

File: data/engines.txt [hunk 1]
```diff
@@ -492,3 +492,16 @@ outfit "AR120 Reverse Thruster"
...
 	"reverse thrusting heat" 3.5
 	description "For pilots who want to be able to pull fancy tricks that most ships are not capable of emulating, a reverse thruster allows a ship to accelerate backwards, without needing to turn around. Because a reverse thruster must be facing forwards, it must be installed in the weapon section of your ship instead of the engine section."
+
+outfit "Stationing Engines"
+	category "Engines"
+	"cost" 75000
+	thumbnail "outfit/ionic afterburner"
+	"mass" 8
+	"outfit space" -10
+	"engine capacity" -8
```
Comment (Bladewood) [inline data/engines.txt]: 8, 10, 8?

Comment (LocalGod79) [inline data/engines.txt]: i had intended for the outfit to use more outfit space than engine capacity - missed the tonnage point during revision - thanks!

**Comment by Pointedstick** [comment]
So both of them are "space brakes?" What's the difference between them supposed to be?

I can have a go at writing clearer descriptions for these outfits once I understand what they do.

**Comment by retrue** [comment]
There are a couple of things that don't sound well in the GBS description.
_holding position in solar system_ should be _holding position in the stelar system_
And this one
_Some captains of especially large ships use it achieve jump velocity faster than turning around using their primary thrusters._
It should be 
_Some captains of especially large ships **who** use it achieve jump velocity faster **that way** than turning around using their primary thrusters._
or
_Some captains of especially large ships use it **to** achieve jump velocity faster than **by** turning around using their primary thrusters._

**Comment by LocalGod79** [comment]
@Pointedstick one replaces the `reverse` key binding (and slows the ship when aligned to the velocity vector), while the other replaces afterburner (and slows when aligned counter to velocity vector) - community feedback indicated that having only one or the other would compromise outfit/play styles about evenly.

@retrue uncommitted changes:
>holding position in a **planetary** system

"stellar system", or "star system" refers only to systems where more than one star is present.  "planetary system" involves any combination of planets and stars.  trying to describe "stopped in space" is apparently as awkward as it is to "achieve"...

> Some captains of especially large ships use it **to** achieve jump velocity faster than turning around **and** using their primary thrusters.

**Comment by LocalGod79** [comment]
sprites are coming together, but i could use a hand with the sound, if anyone has any expertise - i was thinking something like the middle portion of:

https://freesound.org/people/ProfMegavolt/sounds/273357/ for the GBS...

**Comment by DingusShingleton** [comment]
Is this PR still in any way relevant? I hope so.... in addition just thinking it's cool, I created some new stuff for mining which would fit perfectly into independent outfitters.

**Comment by LocalGod79** [comment]
i hope it's relevant - just haven't had much time to work on it lately...

**Comment by DingusShingleton** [comment]
Can I offer my stuff to it? It's one mining ship, two mining weapons, and potentially an outfit (not sure that one fits so much). I don't know if it really stands on its own as an addition, it kind of needs the independent outfitter for distribution.

**Comment by Pointedstick** [comment]
How about this for the descriptions:


description "Stationing Engines are typically used to stabilize space stations' orbits. With this outfit, you can mount a miniature version of the same technology to your own ship, giving it the ability to completely stop. Stationing Engines preclude the use of a normal afterburner."

description `Informally dubbed "the southern Scram", the Graviton Booster System allows equipped starships to make a full stop. Some captains of especially large ships use it achieve jump velocity faster than turning around using their primary thrusters.`

It's not clear yo me why one would choose ever Stationing Engines over the Graviton Booster, though.

**Comment by DingusShingleton** [comment]
Think of stationing engines like bringing a car to a screeching sideways stop. Like this: https://www.youtube.com/watch?v=OEg5Arh-7bw

**Comment by LocalGod79** [comment]
it's a work in progress - a lower price point, wider distribution, and better engine capacity usage were my initial balance points to make it more attractive, but i may not have gone far enough with it...

@DingusShingleton i can add you as a contributor on the fork, if you'd like to add your content to this.

**Comment by DingusShingleton** [comment]
Ok... keep in mind I don't really know what I'm going on git... but I will do my best not to literally ruin everything.

**Comment by DingusShingleton** [comment]
Ok, after realizing that I was going about this the wrong way, I'll just link this here so I can get your thoughts on it before going through the process of committing it all again. These 9 commits are the entirety of the plugin I made. The commit comments have additional context of my thought processes on this stuff. I want to stress that these guns and ship are not good for anything but mining. Most players probably won't give 2 figs about them, but I think they're a blast. Plus asteroid mining can be quite lucrative.

Anyway, sorry I suck at github

https://github.com/endless-sky/endless-sky/compare/master...DingusShingleton:Mining-Outfits

**Pull request closed without merging**